### ISSUE TITLE
PyPI installation package broken

### ISSUE THREAD

**Issue by anantzoid** [Polyconseil/zbarlight#7]
I tried to install using pip on MacOS and Ubuntu and got the same error:

```

    Downloading/unpacking zbarlight
    Downloading zbarlight-1.0.1.tar.gz
    Running setup.py (path:/tmp/pip_build_root/zbarlight/setup.py) egg_info for package zbarlight

    no previously-included directories found matching 'docs'
    no previously-included directories found matching 'tests'
    warning: no previously-included files matching '.py[cod]' found anywhere in distribution
    warning: no previously-included files matching '__pycache__' found anywhere in distribution
    warning: no previously-included files matching '.so' found anywhere in distribution
    warning: no previously-included files found matching 'Makefile'
    warning: no previously-included files found matching 'requirements-dev.txt'
    warning: no previously-included files found matching 'tox.ini'
Downloading/unpacking Pillow (from zbarlight)
  Downloading Pillow-3.2.0.zip (10.5MB): 10.5MB downloaded
  Running setup.py (path:/tmp/pip_build_root/Pillow/setup.py) egg_info for package Pillow
    Single threaded build, not installing mp_compile:1 processes

    warning: no files found matching '*.sh'
    no previously-included directories found matching 'docs/_static'
    warning: no previously-included files found matching '.coveragerc'
    warning: no previously-included files found matching '.editorconfig'
    warning: no previously-included files found matching '.landscape.yaml'
    warning: no previously-included files found matching 'appveyor.yml'
    warning: no previously-included files found matching 'build_children.sh'
    warning: no previously-included files found matching 'tox.ini'
    warning: no previously-included files matching '.git*' found anywhere in distribution
    warning: no previously-included files matching '*.pyc' found anywhere in distribution
    warning: no previously-included files matching '*.so' found anywhere in distribution
Installing collected packages: zbarlight, Pillow
  Running setup.py install for zbarlight
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip_build_root/zbarlight/setup.py", line 57, in <module>
        'Pillow',
      File "/usr/lib/python2.7/distutils/core.py", line 151, in setup
        dist.run_commands()
      File "/usr/lib/python2.7/distutils/dist.py", line 953, in run_commands
        self.run_command(cmd)
      File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command
        cmd_obj.run()
      File "/usr/lib/python2.7/dist-packages/setuptools/command/install.py", line 53, in run
        return _install.run(self)
      File "/usr/lib/python2.7/distutils/command/install.py", line 601, in run
        self.run_command('build')
      File "/usr/lib/python2.7/distutils/cmd.py", line 326, in run_command
        self.distribution.run_command(command)
      File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command
        cmd_obj.run()
      File "/usr/lib/python2.7/distutils/command/build.py", line 128, in run
        self.run_command(cmd_name)
      File "/usr/lib/python2.7/distutils/cmd.py", line 326, in run_command
        self.distribution.run_command(command)
      File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command
        cmd_obj.run()
      File "/usr/lib/python2.7/dist-packages/setuptools/command/build_py.py", line 42, in run
        self.build_packages()
      File "/usr/lib/python2.7/distutils/command/build_py.py", line 373, in build_packages
        self.build_module(module, module_file, package)
      File "/usr/lib/python2.7/dist-packages/setuptools/command/build_py.py", line 60, in build_module
        outfile, copied = _build_py.build_module(self, module, module_file, package)
      File "/usr/lib/python2.7/distutils/command/build_py.py", line 334, in build_module
        "'package' must be a string (dot-separated), list, or tuple")
    TypeError: 'package' must be a string (dot-separated), list, or tuple
    Complete output from command /usr/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip_build_root/zbarlight/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-uowgPO-record/install-record.txt --single-version-externally-managed --compile:
    running install

running build

running build_py

Traceback (most recent call last):

  File "<string>", line 1, in <module>

  File "/tmp/pip_build_root/zbarlight/setup.py", line 57, in <module>

    'Pillow',

  File "/usr/lib/python2.7/distutils/core.py", line 151, in setup

    dist.run_commands()

  File "/usr/lib/python2.7/distutils/dist.py", line 953, in run_commands

    self.run_command(cmd)

  File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command

    cmd_obj.run()

  File "/usr/lib/python2.7/dist-packages/setuptools/command/install.py", line 53, in run

    return _install.run(self)

  File "/usr/lib/python2.7/distutils/command/install.py", line 601, in run

    self.run_command('build')

  File "/usr/lib/python2.7/distutils/cmd.py", line 326, in run_command

    self.distribution.run_command(command)

  File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command

    cmd_obj.run()

  File "/usr/lib/python2.7/distutils/command/build.py", line 128, in run

    self.run_command(cmd_name)

  File "/usr/lib/python2.7/distutils/cmd.py", line 326, in run_command

    self.distribution.run_command(command)

  File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command

    cmd_obj.run()

  File "/usr/lib/python2.7/dist-packages/setuptools/command/build_py.py", line 42, in run

    self.build_packages()

  File "/usr/lib/python2.7/distutils/command/build_py.py", line 373, in build_packages

    self.build_module(module, module_file, package)

  File "/usr/lib/python2.7/dist-packages/setuptools/command/build_py.py", line 60, in build_module

    outfile, copied = _build_py.build_module(self, module, module_file, package)

  File "/usr/lib/python2.7/distutils/command/build_py.py", line 334, in build_module

    "'package' must be a string (dot-separated), list, or tuple")

TypeError: 'package' must be a string (dot-separated), list, or tuple

```

**Comment by rbarrois** [comment]
Hi,

Which version of ``setuptools`` are you using?

**Comment by anantzoid** [comment]
I'm using `Setuptools Version: 20.2.2` and `Pip Version: 8.1.2`

**Comment by rbarrois** [comment]
Could you try with a more recent version of setuptools? It seems that we have issues with ``setuptools<22.0``

**Comment by lfyzjck** [comment]
+1

**Comment by fbochu** [comment]
It should be fixed by https://github.com/Polyconseil/zbarlight/commit/d3403e5da3941963c48e0fa1ae2a045aeb4ffba3. I have published the 1.0.2 release on PyPI.

I close the issue for now (Installation works for me on Debian Jessie and setuptools 5.5.1), but if the problem is still here on your OS, feel free to reopen it.
